apps/etl-func/src/domain/interfaces/tokenizer.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+
+export interface TokenizerClient {
+  tokenize(pii: string): Promise<string>;
+}
+
+const messageTokenizedSchema = z.object({
+  fiscalCode: z.string().uuid().min(1),
+});
+
+export type MessageTokenized = z.infer<typeof messageTokenizedSchema>;
```

Comment:
we tokenize the fiscalCode that is a property of the Message, not the Message itself

I would call it `MessageWithoutPII` o `MessageWithoutSensitiveData`